ext/reflection/tests/ReflectionGenerator_basic.phpt

```diff
@@ -0,0 +1,96 @@
+--TEST--
+ReflectionGenerator basic test
+--FILE--
+<?php
+
+function foo() {
+	yield;
+}
+
+$gens = [
+	(new class() {
+		function a() {
+		yield from (function(){
+			yield;
+		})();
+	}})->a(),
+	(function() {
+		yield;
+	})(),
+	foo(),
+];
+
+foreach ($gens as $gen) {
+	var_dump($gen);
+
+	$gen->valid(); // start Generator
+	$ref = new ReflectionGenerator($gen);
+
+	var_dump($ref->getTrace());
+	var_dump($ref->getExecutingLine());
+	var_dump($ref->getExecutingFile());
+	var_dump($ref->getExecutingGenerator());
+	var_dump($ref->getFunction());
+	var_dump($ref->getThis());
+}
+
+?>
+--EXPECTF--
+object(Generator)#2 (0) {
+}
+array(1) {
+  [0]=>
+  array(5) {
+    ["function"]=>
+    string(9) "{closure}"
+    ["class"]=>
+    string(%d) "class@anonymous%s"
+    ["object"]=>
+    object(class@anonymous)#1 (0) {
+    }
+    ["type"]=>
+    string(2) "->"
+    ["args"]=>
+    array(0) {
+    }
+  }
+}
```

Comment:
Shouldn't this trace have one entry for the inner closure and one for the anon::a()?